deploy/0_preconditions.ts

```diff
@@ -0,0 +1,14 @@
+import { Logger } from 'ethers/lib/utils';
+import { validateDeployment, configureDeploymentSettings } from '../utils/deploy';
+import { LogLevel } from '@ethersproject/logger';
```

Comment:
missing from project dependencies

Reply:
The ethers umbrella project bring this in: https://www.npmjs.com/package/ethers/v/5.6.4

Reply:
It's [extraneous](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-extraneous-dependencies.md)
![image](https://user-images.githubusercontent.com/18407013/167227194-e1c941b3-d20d-4f82-9017-ca1b33a82ba6.png)


I think the recommended way to use this is 

```ts
Logger.setLogLevel(Logger.levels.DEBUG);
```